AUTHORS

```diff
@@ -364,7 +365,7 @@ Patrice Chalin                          chalin@cncf.io
 Patrice Peterson                        patrice.peterson@mailbox.org
 Patrick Mahoney                         pmahoney@greenkeytech.com
 Patrik Cyvoct                           patrik@ptrk.io
-Paul Chaignon                           paul@cilium.io
+Paul Chaignon                           paul.chaignon@gmail.com
```

Comment:
cc @pchaigno, not sure if this was intended

Reply:
Yeah, that's fine.